packages/ui/lit-grid/src/position.ts

```diff
@@ -2,58 +2,35 @@
 // Copyright 2024 DXOS.org
 //
 
-export type CellPosition = { i: number; j: number };
+export type PhysicalPosition = { c: number; r: number };
```

Comment:
c/r is column/row?

Reply:
Yes, though this is an earlier implementation that could use improvement. There is `DxAxis` which could be this object’s keys.

Reply:
Actually very little of the code in this file got used, I’m removing it.